File: nodejs/src/model/Business.js
import ApiClient from '../ApiClient';
import Address from './Address';

export default class Business {
    constructor() {
        this.Address = new Address();
        this.Name = '';
        this.Phone = '';
        this.Fax = '';
        this.WebSite = '';
    }

    static constructFromObject(data, obj) {
        if (data) {
            obj = obj || new Business();

            if (Object.prototype.hasOwnProperty.call(data, 'Address')) {
                obj['Address'] = Address.constructFromObject(data['Address']);
            }
            if (Object.prototype.hasOwnProperty.call(data, 'Name')) {
                obj['Name'] = ApiClient.convertToType(data['Name'], 'String');
            }
            if (Object.prototype.hasOwnProperty.call(data, 'Phone')) {
                obj['Phone'] = ApiClient.convertToType(data['Phone'], 'String');
            }
            if (Object.prototype.hasOwnProperty.call(data, 'Fax')) {
                obj['Fax'] = ApiClient.convertToType(data['Fax'], 'String');
            }
            if (Object.prototype.hasOwnProperty.call(data, 'WebSite')) {
                obj['WebSite'] = ApiClient.convertToType(data['WebSite'], 'String');
            }
        }
        return obj;
    }
}

File: csharp/src/IO.Swagger/Model/Business.cs
using System;
using System.Linq;
using System.IO;
using System.Text;
using System.Text.RegularExpressions;
using System.Collections;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Runtime.Serialization;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using System.ComponentModel.DataAnnotations;
using SwaggerDateConverter = IO.Swagger.Client.SwaggerDateConverter;

namespace IO.Swagger.Model
{
    [DataContract]
    public partial class Business :  IEquatable<Business>, IValidatableObject
    {
        public Business(Address address = default(Address), string name = default(string), string phone = default(string), string fax = default(string), string webSite = default(string))
        {
            this.Address = address;
            this.Name = name;
            this.Phone = phone;
            this.Fax = fax;
            this.WebSite = webSite;
        }

        [DataMember(Name="Address", EmitDefaultValue=false)]
        public Address Address { get; set; }

        [DataMember(Name="Name", EmitDefaultValue=false)]
        public string Name { get; set; }

        [DataMember(Name="Phone", EmitDefaultValue=false)]
        public string Phone { get; set; }

        [DataMember(Name="Fax", EmitDefaultValue=false)]
        public string Fax { get; set; }

        [DataMember(Name="WebSite", EmitDefaultValue=false)]
        public string WebSite { get; set; }
    }
}
